Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1377,6 +1377,20 @@ extension BrowserViewController {
 }
 
 extension BrowserViewController: URLBarDelegate {
+    
+    func showTabTray() {
+        self.webViewContainerToolbar.isHidden = true
+        updateFindInPageVisibility(visible: false)
+        
+        let tabTrayController = TabTrayController(tabManager: tabManager, profile: profile, tabTrayDelegate: self)
+        
+        if let tab = tabManager.selectedTab {
+            screenshotHelper.takeScreenshot(tab)
+        }
+        
+        self.navigationController?.pushViewController(tabTrayController, animated: true)
+        self.tabTrayController = tabTrayController
```

Comment:
nits: `self.` and maybe this assignment not needed?

Reply:
hold the phone, the local tabTrayController variable seems to be needed. I was curious because I saw the old code did this also, and changed it locally, and pushViewController crashes. /me what is going on here?